packages/storybook-builder-vite/package.json

```diff
@@ -24,6 +24,6 @@
         "vue-docgen-api": "^4.40.0"
     },
     "peerDependencies": {
-        "vite": "^2.4.1"
+        "vite": "2.5.10"
```

Comment:
This should be `^2.5.10`, this now locks the peerDependency to 2.5.10
So you can only use this builder with vite 2.5.10 or force the dependency